egs/wsj/s5/steps/nnet3/components.py

```diff
@@ -245,6 +304,12 @@ def AddFinalLayer(config_lines, input, output_dim,
     prev_layer_output = AddAffineLayer(config_lines,
             final_node_prefix , input, output_dim,
             ng_affine_options)
+    if final_mixture_window_length is not None:
+        [prev_layer_output, cur_left_context, cur_right_context] = AddPerDimAffineLayer(config_lines,
```

Comment:
The number of arguments has crossed 3, so it would be safer to use named arguments in the function call.